### DISCUSSION TITLE
Are local structs supported?

### DISCUSSION THREAD

**Discussion by JodiTheTigger** [vlang/v#11712] [Ideas]
**V version: V 0.1.29 35cbca9** 
**OS: windows 10** 

**What did you do?**
```vlang
fn thing()
{
	struct Local_Struct
	{
		test string
	}

	println('thing')
}
```

**What did you expect to see?**
either:
* compiles fine
* error saying *local structs are _not_ supported* (why?)

 
**What did you see instead?**
```
error: expr(): bad token `struct`
    1 | fn thing()
    2 | {
    3 |     struct Local_Struct
      |     ~~~~~~
    4 |     {
    5 |         test string
```

**Comment by medvednikov** [comment]
I've never been a fan of this or used this, but this will be supported, like in Go.  Defining methods won't be possible either.
